indico/modules/events/abstracts/controllers/abstract.py

```diff
@@ -45,7 +51,10 @@ def _process(self):
         defaults = FormDefaults(self.abstract, attachments=self.abstract.files, **custom_field_values)
         form = abstract_form_class(obj=defaults, abstract=self.abstract, event=self.event, management=self.management)
         if form.validate_on_submit():
-            update_abstract(self.abstract, *get_field_values(form.data))
+            fields, custom_fields = get_field_values(form.data)
+            if self.abstract.state == AbstractState.invited:
+                fields['state'] = AbstractState.submitted
```

Comment:
Is this correct? I guess if the user uses the link to submit their invited abstract it never goes through this RH - but if a manager edits the invited abstract (before it has been submitted), it should not suddenly become submitted!